app/views/welcome/index.html.erb

```diff
@@ -1,6 +1,11 @@
 <h1>iStory</h1>
 <p>An interactive journey</p>
 <p></p>
-<div class="tale"><h1><%= link_to 'Tale' %></h1>
-  <p>Lorem ipsum dolor sit amet</p>
+
+<div class="stories">
```

Comment:
I can't see any css class defined for `stories` If you don't need one, please use `id`rather than `class`